### PR TITLE
Template Manager v1

### PR DESCRIPTION
**Pull request by BadWolf42**
TODO: Add more _Equipment/eqLogic_ informations

### CODE REVIEW

File: core/class/jMQTT.class.php [hunk 1]
```diff
@@ -130,6 +141,83 @@ public static function templateParameters($_template = ''){
...
+		foreach ($existing_files as list($n, $f))
+			if ($f == $_filename) {
+				$exists = true;
+				break;
+			}
+		if (!$exists)
+			return false;
+		return unlink(dirname(__FILE__) . '/../../../../' . $_filename);
+	}
+
```
Comment (Domochip) [inline core/class/jMQTT.class.php:215]: Warning, all Jeedom SA plugins using template always refer to name in the json and never to the filename.
It is slower because it parses all template files one by one to the one with the right name inside but ...
... there should be a reason ... maybe ?

Comment (BadWolf42) [reply core/class/jMQTT.class.php:215]: Yes, that's the way I did it first (see https://github.com/BadWolf42/jMQTT/blame/c29b0ad54af96b2823fd886e92309b7d7aa22857/core/class/jMQTT.class.php#L178-L201)
But, when importing a template file with existing template name, I got multiple times the same template name in the list and could not easily delete just a specific one...
So I implemented a file name based delete and planned to use filename instead of template name all over jMQTT.

I don't see any advantage to use template name and don't understand why I should be preferred.

File: core/ajax/jMQTT.ajax.php [hunk 2]
```diff
@@ -65,8 +65,22 @@
 
     ajax::init();
 
+    if (init('action') == 'getOldTemplateList') {
```
Comment (Domochip) [inline core/ajax/jMQTT.ajax.php]: Why do we keep get**Old**TemplateList?

File: core/ajax/jMQTT.ajax.php [hunk 3]
```diff
@@ -65,8 +65,22 @@
...
+        ajax::success(jMQTT::templateParameters(init('template')));
```
Comment (Domochip) [inline core/ajax/jMQTT.ajax.php]: Does jMQTT::templateParameters still necessary?

Comment (BadWolf42) [reply core/ajax/jMQTT.ajax.php]: Arf, I should have cleaned that too.
I will rework the js that is currently using this "old" function and update the PR

File: core/ajax/jMQTT.ajax.php (hunk 2 above)
Comment (BadWolf42) [reply core/ajax/jMQTT.ajax.php]: Removed in commit https://github.com/Domochip/jMQTT/pull/112/commits/2d65645dcec8b1e5464e4f5c01e4f8328054189c

File: core/ajax/jMQTT.ajax.php (hunk 3 above)
Comment (BadWolf42) [reply core/ajax/jMQTT.ajax.php]: Removed in commit https://github.com/Domochip/jMQTT/pull/112/commits/2d65645dcec8b1e5464e4f5c01e4f8328054189c

**Comment by Domochip** [comment]
Well done !!! 👍
